### PR TITLE
Equipment Rework - CR General and Magic Items

### PR DESCRIPTION
**Pull request by Llisandur**
Continuation of the rework that brings the rest of the Core Rulebook items in line with the changes to the weapons and armor.

### CODE REVIEW

File: code/testsuite/csheets/pf_Cleric.xml [hunk 1]
```diff
@@ -1566,9 +1540,35 @@ BIO
...
+			</size>
+			<special_properties></special_properties>
+			<type>GOODS.TOOLS.SPELL COMPONENT.DIVINE FOCUS.HOLY SYMBOL</type>
+			<weight>1</weight>
+			<bonuslist></bonuslist>
+		</item>
 		<total>
 			<weight>59.5 lbs.</weight>
-			<value>926.6 gp</value>
+			<value>901.6 gp</value>
```
Comment (LegacyKing) [inline code/testsuite/csheets/pf_Cleric.xml]: Why did this value change on a test character?

File: code/testsuite/csheets/pf_Cleric.xml [hunk 2]
```diff
@@ -1701,14 +1701,14 @@ BIO
 				<bonuslist></bonuslist>
 			</item>
 			<item>
-				<name>Blanket (Winter)</name>
+				<name>Blanket</name>
```
Comment (LegacyKing) [inline code/testsuite/csheets/pf_Cleric.xml]: Uh, Winter Blanket becomes Blanket?

File: code/testsuite/csheets/pf_Paladin.xml [hunk 3]
```diff
@@ -1088,8 +1088,8 @@ BIO
 		<weapon>
 			<common>
 				<name>
-					<short>Oil (1 Pint Flask)</short>
-					<long>Oil (1 Pint Flask)</long>
+					<short>Oil</short>
+					<long>Oil</long>
```
Comment (LegacyKing) [inline code/testsuite/csheets/pf_Paladin.xml]: We no longer designating how much as per the CR?

File: code/testsuite/csheets/pf_Paladin.xml [hunk 4]
```diff
@@ -1767,7 +1767,7 @@ BIO
 				<long>Medium</long>
 				<short>M</short>
 			</size>
-			<special_properties>+5 alchemical bonus to Fortitude saves against poison for 1 hour</special_properties>
+			<special_properties></special_properties>
```
Comment (LegacyKing) [inline code/testsuite/csheets/pf_Paladin.xml]: Is this intentional?

File: code/testsuite/csheets/pf_Paladin.xml [hunk 5]
```diff
@@ -2346,7 +2346,7 @@ BIO
 		</item>
 		<total>
 			<weight>98 lbs.</weight>
-			<value>7563.45 gp</value>
+			<value>7538.45 gp</value>
```
Comment (LegacyKing) [inline code/testsuite/csheets/pf_Paladin.xml]: Value changes?

File: code/testsuite/csheets/pf_Cleric.xml (hunk 1 above)
Comment (Llisandur) [reply code/testsuite/csheets/pf_Cleric.xml]: The amount off is the value of a silver holy symbol (25 gp). I was attempting to reduce the holy symbol to a single mod-required item, but the migration file doesn't allow EQMODs to be added. I thought I had fixed it, but apparently not.

File: code/testsuite/csheets/pf_Paladin.xml (hunk 5 above)
Comment (Llisandur) [reply code/testsuite/csheets/pf_Paladin.xml]: The amount off is the value of a silver holy symbol (25 gp). I was attempting to reduce the holy symbol to a single mod-required item, but the migration file doesn't allow EQMODs to be added. I thought I had fixed it, but apparently not.

File: code/testsuite/csheets/pf_Cleric.xml (hunk 2 above)
Comment (Llisandur) [reply code/testsuite/csheets/pf_Cleric.xml]: It's listed as "winter blanket" in the CR as a leftover from 3.5, then there's a lighter weight and cheaper cost item simply called "blanket" in the APG. Then the item "blanket" is listed in UE with the original weight and cost as the "winter blanket" with the description from the APG.

File: code/testsuite/csheets/pf_Cleric.xml [hunk 6]
```diff
@@ -1458,14 +1432,14 @@ BIO
 				<short>M</short>
 			</size>
 			<special_properties>Cures 1d8+1 points of damage</special_properties>
-			<type>MAGIC.POTION.CONSUMABLE</type>
+			<type>MAGIC.POTION.CONSUMABLE.ENHANCEMENT</type>
```
Comment (skylarhiebert) [inline code/testsuite/csheets/pf_Cleric.xml]: Are one time recovery items really enhancements, since they have no lasting effect?

Comment (Llisandur) [reply code/testsuite/csheets/pf_Cleric.xml]: That gets added by the EQMOD "Special Ability ~ Spell Effect / Single Use".

File: code/testsuite/csheets/pf_Cleric.xml (hunk 2 above)
Comment (LegacyKing) [reply code/testsuite/csheets/pf_Cleric.xml]: Okay, and is this test unit pulling from the CR, the APG or the UE? If it's the CR, and the UE isn't loaded, then the CR name should be prominent.

Comment (Llisandur) [reply code/testsuite/csheets/pf_Cleric.xml]: Well the cleric test loads both Core Rulebook and Advanced Player's Guide. Which version is it supposed to have?

Comment (Llisandur) [reply code/testsuite/csheets/pf_Cleric.xml]: Also, it's a pain to nearly impossible to change the display name. There really should be a MOD tag to change it.

**Comment by LegacyKing** [comment]
Please fix conflicts, and then I'll review this again. Thanks.

**Comment by Llisandur** [comment]
I'm fairly sure the conflicts are because this is now a few commits behind. What would be the best way to resolve them?

**Pull request merged**